source/_integrations/wallbox.markdown

```diff
@@ -44,6 +44,7 @@ The {% term integration %} adds the following sensors:
 The {% term integration %} adds the following number entities:
 
 - Max charging current
+- Max ICP current; this is the maximum current available for load balancing.
 - Energy price
```

Comment:
I realize the units are not specified for the sensors, either. But I think it could be helpful for the docs.
Current is in A?
and the energy price in cents per kWh?

Reply:
yes, current is in A. energy price is {insert the currency of you country here} / kWh. Do we have a fixed way of adding units in the documentation (for example, after the sensor in brackets or something?)

Reply:
I don't think there is a fixed way. But in brackets after the sensor is common.

Reply:
I added the units, in some cases (sensors) there is no fixed unit or I don't know it (for example, my car doesnt report charging speed, so I have never seen it in practice).